tst/com/amazon/corretto/crypto/provider/test/ConcatenationKdfTest.java

```diff
@@ -58,9 +60,9 @@ public void secretLengthCannotBeZero() {
   @Test
   public void outputLengthCannotBeZeroOrNegative() {
     assertThrows(
-        IllegalArgumentException.class, () -> new ConcatenationKdfSpec(new byte[0], 0, "name"));
+        IllegalArgumentException.class, () -> new ConcatenationKdfSpec(new byte[10], 0, "name"));
```

Comment:
why did we need to update this test?

Reply:
When I manually inspected the coverage report, I noticed that [this line](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/src/com/amazon/corretto/crypto/provider/ConcatenationKdfSpec.java#L33) was not covered and this unit test was suppose to cover it. The test was passing because the secret length was zero, but that was not suppose to be the reason to throw the `IllegalArgumentException`.